javascript/javascript1/week3/noteTakingApp.js

```diff
@@ -0,0 +1,37 @@
+//Note taking app
+
+const notes = [];
+
+function saveNote(content, id){
+   return notes.push({content: content, id: id});
```

Comment:
There's a shorthand you can use for objects, when the propert and variable name is the same:
```javascript
{ content, id } // is the same as { content: content, id: id }
```

Reply:
Done